src/api/test/unit/user_test.rb

```diff
@@ -98,6 +98,24 @@ def test_user_requests
     assert_equal 3, users(:fred).nr_of_requests_that_need_work
   end
 
+  def test_update_globalroles
+    user = User.find_by(login: "tom")
+    local_role = Role.create(title: "local_role", global: false)
```

Comment:
Why to use a variable for that role but not for the others? It's unused after adding that role to the user...